_includes/macros/sub_pages.html

```diff
@@ -8,23 +8,29 @@
 
    Builds sub-pages markup when given:
 
-   sub_pages : An array containing sub-pages.
+   sub_pages              : An array containing sub-pages.
+
+   include_title(optional): Boolean indicating whether to include the block title.
 
    ========================================================================== #}
 
-{% macro render( sub_pages ) %}
+{% macro render( sub_pages, include_title=true ) %}
     {% set is_office_page = request.path.find('/offices/') > -1 %}
     {% for page in sub_pages | sort(attribute='order') %}
-        {% set title = page.short_title or page.title  %}
+        {% set title = 'Our Work' if include_title %}
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/01dd47a2574106da6a469140a72c30134746c265